docs/testing.md

```diff
@@ -0,0 +1,50 @@
+# Distributed Testing
+
+Install Ray on at least two nodes. 
+
+https://docs.ray.io/en/latest/ray-overview/installation.html
+
+```shell
+sudo apt install -y python3-pip python3.12-venv
+python3 -m venv venv
+source venv/bin/activate
+pip3 install -U "ray[default]"
+```
+
+## Start Ray Head Node
+
+```shell
+ray start --head --node-ip-address=10.0.0.23 --port=6379 --dashboard-host=0.0.0.0
+```
+
+## Start Ray Worker Nodes(s)
+
+```shell
+ray start --address=10.0.0.23:6379 --redis-password='5241590000000000'
```

Comment:
Did you need to start redis?

Reply:
no, this is built into ray somehow and this is the password that it uses